src/modules/headset/headset_openxr.c

```diff
@@ -778,15 +835,18 @@ static bool openxr_init(HeadsetConfig* config) {
         suggestedBindings[j].binding = path;
       }
 
+      int successProfiles = 0;
       if (count > 0) {
         XR_INIT(xrStringToPath(state.instance, interactionProfilePaths[i], &path));
-        XR_INIT(xrSuggestInteractionProfileBindings(state.instance, &(XrInteractionProfileSuggestedBinding) {
+        int res = (xrSuggestInteractionProfileBindings(state.instance, &(XrInteractionProfileSuggestedBinding) {
           .type = XR_TYPE_INTERACTION_PROFILE_SUGGESTED_BINDING,
           .interactionProfile = path,
           .countSuggestedBindings = count,
           .suggestedBindings = suggestedBindings
         }));
+        if (XR_SUCCEEDED(res)) successProfiles++;
       }
+      XR_INIT(successProfiles);
```

Comment:
Monado breaks on `/interaction_profiles/pico/neo3_controller` interaction profile, so I wanted to test, if at least one profile is loaded. But `int successProfiles = 0;` should be before profiles iteration.

Reply:
Ok, I understand.  Currently we abort initialization if binding suggestion fails for *any* of the interaction profiles, but due to runtime instability the proposal is to change it to abort if *all* of them fail.

What if we made binding suggestion failure a warning instead of an abort?

```c
XrResult result = xrSuggestInteractionProfileBindings(...);

if (XR_FAILED(result)) {
  lovrLog(LOG_WARN, "XR", "Failed to suggest input bindings for %s", interactionProfilePaths[i]);
}
```

Although suggesting the bindings is usually supposed to succeed, it seems okay to proceed without input if they fail, as long as we log it.